app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -119,7 +117,7 @@ class BrowserFragment : LocaleAwareFragment(),
             insets
         }
         sessionManager = TabsSessionProvider.getOrThrow( activity)
-        observer = Observer(this, loadStateListener)
```

Comment:
Change the constructor class Observer(val fragment: BrowserFragment, private val loadStateListener: LoadStateListener) now that we don't use LoadStateListener as well, too?